jscomp/ext/ext_color.mli

```diff
@@ -22,8 +22,23 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
+type color 
+  = Black
+  | Red
+  | Green
+  | Yellow
+  | Blue
+  | Magenta
+  | Cyan
+  | White
+
+type style 
+  = FG of color 
+  | BG of color 
+  | Bold
+  | Dim
 
 (** Input is the tag for example `@{<warning>@}` return escape code *)
-val ansi_of_tag : string -> string 
+val ansi_of_tag : ?style_of_tag:(string -> style list) -> string -> string 
```

Comment:
looks like the ability to inject a style rather than relying or extending https://github.com/chenglou/bucklescript/blob/944d705c6d7faff3fceffae0aef4728036817b52/jscomp/ext/ext_color.ml#L80?